application/backend/.gitignore

```diff
@@ -0,0 +1,9 @@
+# These are some examples of commonly ignored file patterns.
+# You should customize this list as applicable to your project.
+# Learn more about .gitignore:
+#     https://www.atlassian.com/git/tutorials/saving-changes/gitignore
+
+# Front-end files generated by ui.frontend
+/src/main/resources/**/web_root/*
+/src/main/resources/apps/luna/author/*
+/src/main/resources/apps/bulma/author/*
```

Comment:
Let's improve what we got in howlite and bulma projects. In those projects, we have to compile a frontend module and add some things to `application/backend` so we have to build them together. 
Let's start with moving this to `application/frontend`. We won't need that here after other changes.

Reply:
And we can replace 
```
/src/main/resources/apps/luna/author/*
/src/main/resources/apps/bulma/author/*
```
with 
```
/src/main/resources/**/author/*
```